src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,130 @@
+
+		// Hide basic form on load, show chat bubble instead
+		$selector.fadeIn( "slow" );
+
+		// Add some white space over the footer for the bubble to sit
+		$footer.addClass( componentName + "-mrgn" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+
+				// Equals to bubble default bottom value in CSS
+				var bottomY = $( "main p" ).height();
+				var $window = $( window );
+
+				if ( $window .scrollTop() >= $document.outerHeight() - $window .outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $document.outerHeight() - $window.outerHeight() - $window.scrollTop() ) + bottomY )
+					} );
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+			$( "main" ).after( "<footer id=\"wb-info\" class=\"" + componentName + "-mrgn" + "\"></footer>" );
+		}
+
+		// Close notification aside bubble
+		$( ".notif-close", $selector ).on( "click", function( event ) {
+			event.preventDefault();
+			$( this ).parent().hide();
+			$selector.focus();
+
+			// Do not show notification on next load
+			sessionStorage.setItem( componentName + "-notif", 1 );
+		} );
+	},
+
+
+/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+		if ( elm ) {
+			$elm = $( elm );
+
+			var $bubbleElm = $( "<div class='" + componentName + "-wrap' id=\"adafds\"></div>" );
+			$elm.wrap( $bubbleElm );
+
+			var li = document.createElement( "li" );
+			li.className = "wb-slc";
+
+			// Append the Basic HTML version link version
+			var data_wb_doaction_json = JSON.parse( $elm.attr( "data-wb-doaction" ) );
+
+			if ( data_wb_doaction_json ) {
+				li.innerHTML = "<button  data-wb-doaction='{ \"action\": \"open\", \"source\": " + "\"" + data_wb_doaction_json.source + "\"" + " }' class=\"wb-sl\" >" + $elm.text() + "</button>";
+			}
+
+			// Add linke to the top page skip link.
+			var list = document.getElementById( "wb-tphp" );
+			list.insertBefore( li, list.childNodes[ 0 ] );
+
+			// Initiate chat wizard bubble
+			adjustBubbleMargin( $elm.parent() );
+
+			// Identify that initialization has completed
+			wb.ready( $elm, componentName );
+		}
+	};
+
+// Open overlay from the notification message
+$( ".notif", selector ).on( "click", function() {
+	var $link = $( selector + "-link" );
+	$link.click();
+} );
```

Comment:
The objective of this is to prevent the annoying notification bubble animation to be repeated everything the page is refreshed.